src/components/Nav/Navigation.tsx

```diff
@@ -122,8 +122,8 @@ class Navigation extends React.Component<PropsType, NavigationState> {
 
 const mapStateToProps = (state: KialiAppState) => ({
   navCollapsed: state.userSettings.interface.navCollapse,
-  jaegerUrl: state.jaegerState ? state.jaegerState.jaegerURL : '',
-  jaegerIntegration: state.jaegerState ? state.jaegerState.integration : false
+  jaegerUrl: state.jaegerState ? state.jaegerState.url : '',
+  jaegerIntegration: state.jaegerState ? state.jaegerState.enabled : false
 });
```

Comment:
I prefer to move this to jaegerEnabled if we are going to remove the integration check.